`General`: Add README to data export (#6851)
ls1intum/Artemis: src/main/resources/templates/dataexport/README.md

# Art. 15 GDPR README

## Purposes of processing
Teaching and examinations as a legal duty that is assigned to us by the legislator.

## Categories of personal data
Only general personal data, no special categories.

> ## Recipients or categories of recipients to whom the personal data have been or will be disclosed, in particular recipients in third countries or international organisations

Reviewer: ```suggestion

## Recipients or categories of recipients to whom the personal data have been or will be disclosed, in particular recipients in third countries or international organisations
```

> Teaching assistants and instructors, who are involved in the teaching and examination process.

Reviewer: ```suggestion
Teaching assistants and instructors who are involved in the teaching and examination process.
```

## Data storage period
The data is stored for the duration of the legal retention period.

## Your rights

>  - You have the right to request from the controller rectification or erasure of personal data or restriction of processing concerning the data subject or to object to processing according to Art. 15 1e) GDPR.

Reviewer: ```suggestion
 - You have the right to request rectification or erasure of personal data or restriction of processing concerning the data subject or to object to processing according to Art. 15 1e) GDPR.
```
The `from the controller` feels out of place in the sentence structure. I think it would be clearer without.

Reviewer 2: If the info is needed for legal reasons, maybe re-write it to something like this?

```suggestion
 - You have the right to request the controller to rectify or erase your personal data, restrict its processing, or object to its processing. This is according to Article 15, clause 1e) of the General Data Protection Regulation (GDPR).
```

Author: That's the wording in the original legal text, therefore I'd like to leave it like that.
![image](https://github.com/ls1intum/Artemis/assets/84102468/c6ce96f1-65ad-41df-8627-4c1ee6bb18df)

# Art. 15 GDPR README

## Purposes of processing
Teaching and examinations as a legal duty that is assigned to us by the legislator.

## Categories of personal data
Only general personal data, no special categories.

## Recipients or categories of recipients to whom the personal data have been or will be disclosed, in particular recipients in third countries or international organisations

> Teaching assistants and instructors who are involved in the teaching and examination process.

Reviewer 3: Administrators also have access to this data
